docs/tutorials.rst

```diff
@@ -1,5 +1,153 @@
 Tutorials
```

Comment:
@wlav, could you take a look and give feedback on our documentation intent to capture the cppyy/python interop use case as motivation for CppInterOp?

Reply:
? What specific piece of text needs commenting?

Reply:
@wlav, basically this file: `docs/tutorials.rst`. I'd like to make sure we are not making some unreasonable claim.